### PR TITLE
[max] Update Readme

### PR DESCRIPTION
**Pull request by chiefymuc**
- Correct Thing definition example within Bridge (current example works but throws errors)
- Include example for Shutter contact
- Update Item definition example with shutter contact and UoM

### CODE REVIEW

**Comment by martinvw** [comment]
It would be great if you could include a `Signed-off-by` (to certify that you wrote it or otherwise have the right to pass it on as an open-source), see http://docs.openhab.org/developers/contributing/contributing.html#sign-your-work

If running a local git client you can use `git commit --amend --signoff`

File: addons/binding/org.openhab.binding.max/README.md [hunk 1]
```diff
@@ -82,8 +83,12 @@ Group gMAX 			"MAX Heating" 	<temperature>	[ "home-group" ]
 
 Switch maxBattery "Battery Low" (gMAX) {channel="max:thermostat:KEQ0565026:KEQ0648949:battery_low"}
 String maxMode    "Thermostat Mode Setting" (gMAX) {channel="max:thermostat:KEQ0565026:KEQ0648949:mode"}
-Number maxActual  "Actual measured room temperature  [%.1f °C]" (gMAX) {channel="max:thermostat:KEQ0565026:KEQ0648949:actual_temp"}
-Number maxSetTemp "Thermostat temperature setpoint [%.1f °C]" (gMAX) {channel="max:thermostat:KEQ0565026:KEQ0648949:set_temp"}
+Number:Temperature maxActual  "Actual measured room temperature  [%.1f %unit%]" (gMAX) {channel="max:thermostat:KEQ0565026:KEQ0648949:actual_temp"}
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.max/README.md]: What is the intention to use `Number:Temperature` here? The binding does not support UoM until #3506 will be merged.

**Comment by chiefymuc** [comment]
I've updated the commit, but how do I delete the first one? I did everything in the online editor and this is my first time contributing on github.

File: addons/binding/org.openhab.binding.max/README.md [hunk 2]
```diff
@@ -84,6 +85,10 @@ Switch maxBattery "Battery Low" (gMAX) {channel="max:thermostat:KEQ0565026:KEQ06
 String maxMode    "Thermostat Mode Setting" (gMAX) {channel="max:thermostat:KEQ0565026:KEQ0648949:mode"}
 Number maxActual  "Actual measured room temperature  [%.1f °C]" (gMAX) {channel="max:thermostat:KEQ0565026:KEQ0648949:actual_temp"}
 Number maxSetTemp "Thermostat temperature setpoint [%.1f °C]" (gMAX) {channel="max:thermostat:KEQ0565026:KEQ0648949:set_temp"}
+
+Contact maxShuttercontactState "Contact State" (gMAX) {channel="max:shuttercontact:KEQ0565026:NEQ1150510:contact_state"}
+Switch maxShuttercontactBattery "Contact Battery Low" <battery> (gMAX) {channel="max:shuttercontact:KEQ0565026:NEQ1150510:battery_low"}
+
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.max/README.md]: Please remove the empty line.

File: addons/binding/org.openhab.binding.max/README.md [hunk 3]
```diff
@@ -96,6 +101,7 @@ sitemap demo label="Main Menu"
 			Setpoint item=maxSetTemp minValue=4.5 maxValue=32 step=0.5 icon="temperature"
 			Text item=maxActual  icon="temperature"
 			Switch  item=maxBattery
+
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.max/README.md]: Please remove the empty line.

**Comment by cweitkamp** [comment]
@chiefymuc Are you still with us? Can you incorporate the final changes?

**Comment by martinvw** [comment]
I still do not see signed off commits if you state it here in a message that you do agree and conform to the DCO, we can merge

**Comment by chiefymuc** [comment]
I agree and conform to the DCO. 

Signed-off-by: David Kalwar <david.kalwar@gmail.com> (github: chiefymuc)

**Comment by martinvw** [comment]
Thanks!

**Pull request merged**